src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -760,6 +760,8 @@ sub large_dither_checks {
     my $all_dither = shift;
     my $time_tol = 10;         # Commands must be within $time_tol of expectation
 
+    my $warn_free = 1;
```

Comment:
Great idea, thanks!  I can't think of a way that $self->{warn} could change length during this routine except via issuing one of these warnings, so this makes sense.